libretroshare/src/jsonapi/restbedservice.h

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * libretroshare/src/jsonapi/: restbedservice.h                                *
+ *                                                                             *
+ * libretroshare: retroshare core library                                      *
+ *                                                                             *
+ * Copyright 2019-2019 Cyril Soler                                             *
+ *                                                                             *
+ * This program is free software: you can redistribute it and/or modify        *
+ * it under the terms of the GNU Lesser General Public License as              *
```

Comment:
We should use AGPLv3 at least for the optional components that involve web interactions tecnologies, moreover restbed is relead under AGPL so also RetroShare must comply with AGPL when compiling with this options

Reply:
There's no web interaction is restbedservice. But given that it is in jsonapi/ directory, I agree to use AGPLv3. It's simpler for writing the licence file.